vkubelet/apiserver.go

```diff
@@ -26,6 +27,16 @@ func ApiserverStart(provider Provider) {
 	p = provider
 	certFilePath := os.Getenv("APISERVER_CERT_LOCATION")
 	keyFilePath := os.Getenv("APISERVER_KEY_LOCATION")
+
+	var err error
+	if valid := certs.IsCertKeyPairValid(certFilePath, keyFilePath); !valid {
+		log.Printf("provided cert/key pair is invalid or empty, generating.")
```

Comment:
Should this just error out in this case and the user would have to restart VK without the certs?

This also would need a `\n` or a `log.Println`.

Some minor nits:
If we do keep this it may be good to separate the log entries.
The error message my not be exactly accurate either in case these env vars were not set at all.

Reply:
Also should warn that these certs are not suitable for production and the user should provide their own.

Reply:
👍 both of these are great suggestions.

Reply:
> Should this just error out in this case and the user would have to restart VK without the certs?

@robbiezhang said we should generate them if they were invalid. I'd tend to say it should _fail_ if invalid and not generate over top of them. I'll make that change if we agree that's the right behavior.